_posts/2025-09-29-how-to-connect-ios-to-midi.md

```diff
@@ -0,0 +1,80 @@
+
+Ableton have a nice guide for [Setting up a virtual MIDI network](https://help.ableton.com/hc/en-us/articles/209071169-Setting-up-a-virtual-MIDI-network) which has some details of how it works on different platforms
+
+# Bluetooth MIDI
+
+Bluetooth MIDI is best for connecting direct to other instruments, but can also be used to connect to a computer. It runs on Bluetooth LE so has a small amount of latency, but is one of the easiest options to set up. 
+
+- Open Audio Midi Settings on the Mac
+- Open the Midi Studio Window, then tap the Bluetooth icon in the top right to get the Bluetooth setup window
+- Set your Mac to "Advertise" itself to local devices
+- In Playtonik, tap the settings cog in the top right then tap the Bluetooth icon
+- The system dialogue for device pairing will open, select your Mac and pair it with your iPhone/iPad
+- Close the Bluetooth dialog, then select the Bluetooth option in the MIDI Input/Output dropdown
+- Open Midi Settings in your DAW, you should now see the Bluetooth connection as an available MIDI input/output
+
+<img src="/assets/iOSmidi/BluetoothMidiSetup.png" style="border-radius: 15px" alt=""/>
+
+For a Bluetooth MIDI instrument, follow the pairing process for your device using the Bluetooth dialog in Playtonik.
+
+# Virtual Devices
+
+If you want to try out MIDI, but don't have any external gear, all is not lost! There are other apps for iPhone and iPad that allow you to get connected.
+
+Options on iPhone are more limited as you can only run one foreground app at a time. Playtonik doesn't run in the background (to save on battery life) but some audio apps like the awesome Fugue Machine and Korg Gadget will. You can set them playing, then switch to Playtonik and send/receive MIDI with the background app. On iPad you can run two apps in the foreground with Stage Manager. With the extra screen space you can set up a side by side view of Playtonik with another MIDI enabled app and control both at once.
+
+Fugue machine example:
+- Open Fugue Machine and tap the ... icon in the top right
+- Enable MIDI Out via Settings screen using the "Fugue Machine MIDI Out" option
+- Pick a pattern and press play
+- Open Playtonik and go tap the settings cog icon
+- Select the Fugue Machine MIDI as the device
+- Notes from the Fugue pattern will trigger in Playtonik
+
+<img src="/assets/iOSmidi/VirtualDeviceSetup.png" style="border-radius: 15px" alt=""/>
+
+Note: As of Playtonik 1.5 the app doesn't show up as a virtual MIDI port. Playtonik can connect to other apps that share a virtual port, but not every app does. We have only tested with Fugue Machine, Korg Gadget & Logic Pro (iPad).
+
+# Find out more
+
+If you want to find out more abou Playtonik check out our website [5of12.co.uk](https://www.5of12.co.uk/#Playtonik)
+To see some examples of how we've integrated Playtonik into our music setups check out our [Playtonik Playlist on YouTube](https://www.youtube.com/playlist?list=PL4zuBsU0ueMlLnphaBhRjQ5iLQO9R2qMJ)
```

Comment:
Might want pop another line between these paragraphs to separate them?

<img width="793" height="150" alt="Image" src="https://github.com/user-attachments/assets/acc121f7-90c4-4648-adaf-6eac9413923e" />